From 1a6981aee44b2b43bfd96442de2f8f8376c1f3de Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 1 May 2015 18:22:06 +0800
Subject: [PATCH] MLK-10801 ARM: dts: change wm8960 to codec master mode and
 reconfig pin to avoid sync error for imx6ul-evk

change wm8960 to codec master mode and reconfig BCLK MCLK SYNC pin to avoid
sync error for imx6ul-evk.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index a7d99dc..122877d 100755
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -42,6 +42,7 @@
 		model = "wm8960-audio";
 		cpu-dai = <&sai2>;
 		audio-codec = <&codec>;
+		codec-master;
 		gpr = <&gpr>;
 		/*
                  * hp-det = <hp-det-pin hp-det-polarity>;
@@ -414,11 +415,11 @@
 
 		pinctrl_sai2: sai2grp {
 			fsl,pins = <
-				MX6UL_PAD_JTAG_TDI__SAI2_TX_BCLK	0x130b0
-				MX6UL_PAD_JTAG_TDO__SAI2_TX_SYNC	0x130b0
+				MX6UL_PAD_JTAG_TDI__SAI2_TX_BCLK	0x70b0
+				MX6UL_PAD_JTAG_TDO__SAI2_TX_SYNC	0x70b0
 				MX6UL_PAD_JTAG_TRST_B__SAI2_TX_DATA	0x120b0
 				MX6UL_PAD_JTAG_TCK__SAI2_RX_DATA	0x130b0
-				MX6UL_PAD_JTAG_TMS__SAI2_MCLK		0x130b0
+				MX6UL_PAD_JTAG_TMS__SAI2_MCLK		0x70b0
 				MX6UL_PAD_SNVS_TAMPER4__GPIO5_IO04	0x17059
 			>;
 		};
-- 
1.8.0

